src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php

```diff
@@ -129,16 +129,25 @@ protected function issueDoi(Dataset $dataset, array $loggingContext)
         $this->logger->info('Attempting to issue DOI', $loggingContext);
 
         $doi = $dataset->getDoi();
+
         if ($doi instanceof DOI) {
             $this->logger->warning('The DOI already exist for dataset', $loggingContext);
+            $doiId = $doi->getDoi();
+            try {
+                $doiMetaData = $doiUtil->getDOIMetadata($doiId);
```

Comment:
$doiUtil is undefined in here

Reply:
I think you can just make a new instance of it.